src/components/vm/vmDetailsPage.jsx

```diff
@@ -63,6 +65,10 @@ export const VmDetailsPage = ({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    // not initialized yet?
+    if (!vm.capabilities)
+        return <EmptyStatePanel title={ _("Loading... ") } loading />;
```

Comment:
You have an extra space here, I guess it will not pick up existing translations?

Reply:
Eek, that wasn't intended -- I surely want to use the existing translation, I explicitly grepped for that. Thanks for spotting! I'll let the tests finish and re-push.